web/themes/custom/cloudy/templates/content/page/search-api-page.html.twig

```diff
@@ -1,11 +1,17 @@
 {{ form }}
 
+{% set pathParts = path('<current>')|split('/') %}
+
 {% if no_of_results %}
-  <p class="mt-5 mb-3">{{ no_of_results }}</p>
+  <p class="mt-5 mb-3">{{ no_of_results }}. If you are not seeing result you want, it may not be migrated yet. Try your search on <a href="https://www.portlandoregon.gov/?mode=search&search_action=SearchResults&top_category_tree_id=25777&filter_category_tree_id=25776&search_category_tree_id=25777&submit=Search&search_words={{ pathParts[pathParts|length - 1] }}">PortlandOregon.gov</a>.</p>
 {% endif %}
 
 {{ no_results_found }}
 
+{% if no_of_results == 0 %}
+It may not be migrated yet. Try your search on <a href="https://www.portlandoregon.gov/?mode=search&search_action=SearchResults&top_category_tree_id=25777&filter_category_tree_id=25776&search_category_tree_id=25777&search_words={{ pathParts[pathParts|length - 1] }}&submit=Search">PortlandOregon.gov</a>.
+{% endif %}
+
 {{ search_help }}
 
 {{ results }}
```

Comment:
I didn't get this into the story, but when we talked through this story in sprint planning we talked about adding this message below the results as well. 

Below the results doesn't need the result count repeated, so you can just add "The information you are looking for may not be migrated yet. Try your search on PortlandOregon.gov." in a paragraph element.

Reply:
Updated